qutebrowser/browser/commands.py

```diff
@@ -552,6 +552,12 @@ def _back_forward(self, tab, bg, window, count, forward):
         else:
             widget = self._current_widget()
 
+        if not count and index:
+            count = abs(history.current_idx() - index)
+
+        if not count:
+            count = 1
```

Comment:
I found this a bit hard to follow, and it'll also mean we go back one item when `:back` is used with the current index for some reason.

What about this instead?

```python
if count is None:
    if index is None:
        count = 1
    else:
        count = abs(history.current_idx() - index)
```